biocypher_metta/adapters/gencode_exon_adapter.py

```diff
@@ -76,3 +76,34 @@ def get_nodes(self):
                         print(
                             f'fail to process for label to load: {self.label}, type to load: {self.type}, data: {line}')
 
+    def get_edges(self):
+        with gzip.open(self.filepath, 'rt') as input:
+            for line in input:
+                if line.startswith('#'):
+                    continue
+
+                data_line = line.strip().split()
+                if data_line[GencodeExonAdapter.INDEX['type']] != 'exon':
+                    continue
+
+                info = self.parse_info_metadata(data_line[GencodeExonAdapter.INDEX['info']:])
+                transcript_key = info['transcript_id'].split('.')[0]
+                if info['transcript_id'].endswith('_PAR_Y'):
+                    transcript_key = transcript_key + '_PAR_Y'
+                exon_key = info['exon_id'].split('.')[0]
+                if info['exon_id'].endswith('_PAR_Y'):
+                    exon_key = exon_key + '_PAR_Y'
```

Comment:
Can you also add this part in the `get_nodes` of exon?

Reply:
i wasn't sure if there are any exons that end with PAR_Y and it is not on the sample but i added it just to be safe